tests/mov.cmp0

```diff
@@ -0,0 +1,18 @@
+=== Debug Information ===
+IncrementInstr (1)
+IncrementInstr (1)
+IncrementInstr (1)
+IncrementInstr (1)
+IncrementInstr (1)
```

Comment:
Doesn't make sense per previous comment (there's only five `IncrementInstr` and there's 6 `+`)